README.md

```diff
@@ -4,6 +4,15 @@
 - We are using [Swiftlint](https://github.com/realm/SwiftLint) to enforce Swift style and conventions
 - We're using [PaintCode Plugin for Sketch](https://www.paintcodeapp.com/sketch) for vector image assets.
 
+
+# Quick Start
+* Clone or Download ZIP
+* Build with Xcode, and make sure you've installed [SwiftGen](https://github.com/SwiftGen/SwiftGen/tree/stable?tab=readme-ov-file#installation) and [Swiftlint](https://github.com/realm/SwiftLint).
+* If you encounter the error shown below when building, make sure there are no spaces in the path of the project folder.
```

Comment:
Nice notice, I think these kind of instructions can be organized into the Wiki page. And maybe put the wiki link on the `README`.